src/engine/channelhandle.h

```diff
@@ -219,7 +219,7 @@ class ChannelHandleMap {
         } else {
             // We need to initialize simple types ourselves
             while (m_data.size() < iSize) {
-                m_data.append({});
+                m_data.append(std::move(T()));
```

Comment:
Move is unnecessary since this is already a prvalue iiuc
```suggestion
                m_data.append(T());
```